hubstorage/client.py

```diff
@@ -40,7 +40,7 @@ def _hc_retry_on_exception(err):
 class HubstorageClient(object):
 
     DEFAULT_ENDPOINT = 'https://storage.scrapinghub.com/'
-    USERAGENT = 'python-hubstorage/{0}'.format(__version__)
+    DEFAULT_USER_AGENT = 'python-hubstorage/{0}'.format(__version__)
```

Comment:
@vshlapakov now I'm afraid that property approach not going to work. I've found 2 examples in our code base that don't work with it:

1)

```
In [8]: class HubstorageClientWithCustomUA(HubstorageClient):
   ...:     USERAGENT = 'b'
   ...:

In [9]: hsc = HubstorageClientWithCustomUA()

In [10]: hsc.USERAGENT
Out[10]: 'b'

In [11]: hsc.DEFAULT_USER_AGENT
Out[11]: 'python-hubstorage/0.23.1'

In [12]: hsc.user_agent
Out[12]: 'python-hubstorage/0.23.1'
```

2) 

```
In [13]: class HubstorageClientWithCustomUA(HubstorageClient):
    USERAGENT = HubstorageClient.USERAGENT + "/c"
   ....:
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-13-df6deb570a9f> in <module>()
----> 1 class HubstorageClientWithCustomUA(HubstorageClient):
      2     USERAGENT = HubstorageClient.USERAGENT + "/c"
      3

<ipython-input-13-df6deb570a9f> in HubstorageClientWithCustomUA()
      1 class HubstorageClientWithCustomUA(HubstorageClient):
----> 2     USERAGENT = HubstorageClient.USERAGENT + "/c"
      3

TypeError: unsupported operand type(s) for +: 'property' and 'str'
```

--

Instead we can set `USERAGENT = None` and then in `_create_session`:

```
        s.headers.update({'User-Agent': self.USERAGENT or self.user_agent})
```

and raise deprecation warning if `USERAGENT` is not `None`